drivers/can/can_shell.c

```diff
@@ -68,6 +68,11 @@ static struct k_poll_event can_shell_rx_msgq_events[] = {
 static void can_shell_tx_msgq_triggered_work_handler(struct k_work *work);
 static void can_shell_rx_msgq_triggered_work_handler(struct k_work *work);
 
+static bool can_device_check(const struct device *dev)
```

Comment:
Maybe call this `is_can_and_ready`?

Reply:
I think the existing name is fine as this function may be extended to do further checks in the future.